api/lanes.py

```diff
@@ -608,26 +529,19 @@ class ContributorLane(QueryGeneratedLane):
     ROUTE = 'contributor'
     MAX_CACHE_AGE = 48*60*60    # 48 hours
 
-    def __init__(self, _db, contributor_name, contributor_id=None,
```

Comment:
`contributor_id` isn't used on the endpoint, so it's not possible to use it for the lane.